.github/workflows/azure-static-web-apps-icy-glacier-004ab4303.yml

```diff
@@ -4,14 +4,17 @@ on:
   push:
     branches:
       - master
-  pull_request:

```

Comment:
hi @jafin , I would prefer to keep this, it allow me to test on many devices before merging.
I do not have solution for PR from fork. The best is as you set up bellow to skip it when PR comes from a fork :  !github.event.pull_request.head.repo.fork

Reply:
@guillaume-chervet  All good, was just a suggestion. I was curious if there was an option on PR builds to get the builds to go green, so as a committer it was easy to ensure I didn't break stuff,  But if the way you have it setup now benefits in deployments then that makes sense to maintain it.  I'll close the PR